ix-dev/community/lms/templates/docker-compose.yaml

```
@@ -0,0 +1,26 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.lms_container_name, "image") %}
+
+{% do c1.add_caps(["CHOWN", "SETGID", "SETUID"]) %}
+
+{% do c1.ports.add_port(9000, values.network.web_port) %}
+{% do c1.ports.add_port(9090, values.network.cli_port) %}
+{% do c1.ports.add_port(3483, 3483) %}
+{% do c1.ports.add_port(3483, 3483, {"protocol": "udp"}) %}
```

```suggestion
{% do c1.add_port(values.network.web_port) %}
{# TODO: We might have to expose the container port as well on ui #}
{% do c1.add_port(values.network.cli_port, {"container_port": 9090}) %}
{# FIXME: those should be exposed on UI too, and use the c1.add_port() #}
{% do c1.ports.add_port(3483, 3483) %}
{% do c1.ports.add_port(3483, 3483, {"protocol": "udp"}) %}
```
Left couple todo's. I'll check them later

---

Done, left the TODO/FIXME's in place